docs/index.md

```diff
@@ -1014,6 +1014,60 @@ for all of the CSV files.
 
 ## Advanced Features
 
+
+### Singletons with the "just_once" feature
+
+Snowfakery scales up to larger data volumes
+by evaluating your recipe over and over again.
+Each one is called an `iteration`.
+
+Some objects are meant to be produced only once, regardless
+of how many times the recipe executes. The programming
+language term for this is a "singleton". For example
+an accounting system might generate a dataset that has
+exactly 3 Ledger objects, Checking and Savings. You could have
+dozens of Entries per Ledger or maybe billions. But it
+might want to always generate exactly 3 Ledgers.
+
+Here is how you would do that:
+
+```yaml
+- object: Ledger
+  just_once: True
+  nickname: Checking
+  fields:
+    Name: Checking
+
+- object: Ledger
+  just_once: True
+  nickname: Savings
+  fields:
+    Name: Savings
+
+- object: Entry
+  count: 1000
+  fields:
+    Ledger: Checking
+    ...
+
+- object: Entry
+  count: 1000
+  fields:
+    Ledger: Savings
+    ...
+```
+
+Now if you execute this from the Snowfakery command line like this:
+
+```s
```

Comment:
I don't think the 's' is needed.

Reply:
It seems to format the text as a shell script. I guess I could have called it "sh" to be clearer.

Reply:
Experimentally "s" works best.

    ```s 

```s 
$ foo bar baz 
xyzzy
```

```bash
$ foo bar baz 
xyzzy
```

```sh
$ foo bar baz 
xyzzy
```

works.

Reply:
TIL